server/player/player.go

```diff
@@ -964,6 +989,26 @@ func (p *Player) Respawn() {
 	p.SetVisible()
 }
 
+// SpawnPos returns spawn position in the current world.
+func (p *Player) SpawnPos() cube.Pos {
+	return p.World().PlayerSpawn(p.uuid)
+}
+
+// SetSpawnPos sets spawn position for the player in the provided world.
+func (p *Player) SetSpawnPos(pos cube.Pos, w *world.World) {
+	w.SetPlayerSpawn(p.UUID(), pos)
+}
+
+// realSpawnPos returns position and world where player should be spawned.
+func (p *Player) realSpawnPos() (cube.Pos, *world.World, bool) {
```

Comment:
a better name for this would be great, maybe just something as simple as "spawnPos". Improving the docs to mention what the returned bool indicates would also be great